### PR TITLE
Adding Kosh to conda-forge

### PR DESCRIPTION
**Pull request by doutriaux1**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/kosh/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    url: https://github.com/LLNL/kosh/archive/v{{ version }}.tar.gz
+    sha256: 96a82d9ed402da865741a85fe90eb021bd911864d3d85fe3e3ab273a443868fb
+
+build:
+  skip: True  # [win]
+  number: 0
+  # noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - kosh = kosh.cli.driver:main
```
Comment (chrisburr) [inline recipes/kosh/meta.yaml:18]: It looks like `kosh` isn't an entrypoint? https://github.com/LLNL/kosh/blob/3c4b3f9c332f502721439fd75068346d210f27e2/setup.py#L32
```suggestion
  # Can't be noarch due to the use of "scripts" in setup.py
  skip: True  # [win]
  number: 0
  script: {{ PYTHON }} -m pip install . -vv
```

File: recipes/kosh/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    - python
+    - llnl-sina
+    - networkx
+    - numpy
+
+test:
+  imports:
+    - kosh
+  commands:
+    - pip check
```
Comment (chrisburr) [inline recipes/kosh/meta.yaml:35]: Can the included commands be tested with something like?
```suggestion
    - pip check
    - kosh --help
    - sbang --help
    - init_sina.py --help
```

**Comment by doutriaux1** [comment]
@chrisburr I tihnk it is ready now.

**Comment by doutriaux1** [comment]
@chrisburr although this was merged I do not see anything on conda-forge yet. Any idea what went wrong?

https://github.com/conda-forge/kosh-feedstock

**Comment by doutriaux1** [comment]
@conda-forge-admin @conda-forge-coordinator why is this [assing and merged and in feedstocks but not avaiable on conda-forge?

**Pull request merged**